channel-api/src/main/java/com/synopsys/integration/alert/channel/api/DistributionEventReceiver.java

```diff
@@ -61,12 +66,12 @@ public final String getDestinationName() {
     }
 
     protected void handleAlertException(AlertException e, DistributionEvent event) {
-        logger.error("An exception occurred while handling the following event: {}", event, e);
+        logger.error("An exception occurred while handling the following event: {}. {}", event.getEventId(), e);
```

Comment:
Using `{}` for the exception is unnecessary. When the last parameter is a `Throwable`, SLF4J will log the stack trace. https://www.baeldung.com/slf4j-log-exceptions